### DISCUSSION TITLE
Mão na massa: Criando uma calculadora

### DISCUSSION THREAD

**Discussion by arthur221** [balta-io/forum#36] [Tira Dúvidas] [answered]
Boa tarde Balta.

Estava fazendo sua aula em fundamentos de C# e me deparei com um erro neste código:
float v1 = float.Parse(Console.ReadLine()); 
O erro está dando no Console.ReadLine 
A mensagem diz: "Possível argumento de referência nula para o parâmetro 's' em 'float float.Parse(string s)'.", você poderia me ajudar?

Grato Arthur.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @arthur221, como vai?

Esta mensagem é apenas um `warning`(Um aviso) dizendo que pode ser que o valor que está manipulando seja nulo.
Isto não interfere neste exemplo e sua aplicação deve compilar/funcionar normalmente.
